### PR TITLE
Fixes 284: No permissions on initial Add content screen

### PR DESCRIPTION
**Pull request by Andrewgdewar**
Screen Shot 2022-11-15 at 11 22 25 AM
The above button was not being disabled when write permissions were not present.

### CODE REVIEW

File: src/components/ConditionalTooltip/ConditionalTooltip.tsx [hunk 1]
```diff
@@ -9,7 +9,9 @@ interface Props extends TooltipProps {
 const ConditionalTooltip = ({ show, children, setDisabled, ...rest }: Props) =>
   show ? (
     <Tooltip {...rest}>
-      {children && cloneElement(children, setDisabled ? { isDisabled: setDisabled } : undefined)}
+      <div>
```
Comment (Andrewgdewar) [inline src/components/ConditionalTooltip/ConditionalTooltip.tsx:12]: This fixes an issue where sometimes the tooltip won't show up (this is a known issue with the tooltip in patternfly).

File: src/components/ContentListTable/components/EmptyTableState.tsx [hunk 2]
```diff
@@ -32,7 +35,7 @@ const EmptyTableState = ({ notFiltered, clearFilters }: Props) => {
           ? 'No custom repositories'
           : 'No custom repositories match the filter criteria'}
       </Title>
-      <EmptyStateBody>
+      <EmptyStateBody className={classes.emptStateBody}>
```
Comment (Andrewgdewar) [inline src/components/ContentListTable/components/EmptyTableState.tsx:38]: This fixes a padding issue I noticed with the button and the accompanying text above it.

**Comment by jlsherrill** [comment]
https://issues.redhat.com/browse/HMSCONTENT-284

**Comment by swadeley** [comment]
Hi

I confirm, with `is_org_admin: false` I now see that the Add Repo button is disabled and tooltip shows "You do not have the required permissions to perform this action."

Thank you

**Review by swadeley — APPROVED** [review]
ACK

**Pull request merged**